src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -1094,19 +1098,17 @@ namespace HighTempRadiantSystem {
         HighTempRadSysOn = false;
 
         // If this was never allocated, then there are no radiant systems in this input file (just RETURN)
-        if (!allocated(state.dataHighTempRadSys->QHTRadSrcAvg)) return;
+        if (state.dataHighTempRadSys->NumOfHighTempRadSys == 0) return;
 
         // If it was allocated, then we have to check to see if this was running at all...
         for (int RadSysNum = 1; RadSysNum <= state.dataHighTempRadSys->NumOfHighTempRadSys; ++RadSysNum) {
-            if (state.dataHighTempRadSys->QHTRadSrcAvg(RadSysNum) != 0.0) {
-                HighTempRadSysOn = true;
-                break; // DO loop
-            }
+            auto &thisHTR = state.dataHighTempRadSys->HighTempRadSys(RadSysNum);
+            thisHTR.QHTRRadSource = thisHTR.QHTRRadSrcAvg;
+            if (thisHTR.QHTRRadSrcAvg != 0.0) HighTempRadSysOn = true;
         }
```

Comment:
Another example, where do you see loop counter RadSysNum, only in the creation of the reference.

Reply:
+1